src/pushcollector/_impl/collector.py

```diff
@@ -20,8 +20,9 @@ def update_push_items(self, items):
         """Record a state change on one or more push items.
 
         Arguments:
-            items (List[dict])
-                    A list of push item dicts.
+            items (List[dict], List[:class:`~pushsource.PushItem`])
```

Comment:
Have you checked if this renders correctly when docs are generated?

In order for the pushsource.PushItem class to be resolved and hyperlinked correctly, I think you would typically need to add an entry to intersphinx_mapping in docs/conf.py.

Reply:
It compiled properly using make but the link to pushsouce.PushItem didn't work as expected.
However, there was an error rendering this in tox environment.
Updating intersphinx_mapping works as expected.